src/cmds/pbs_rsub.c

```diff
@@ -100,7 +102,7 @@ process_opts(int argc, char **argv, struct attrl **attrp, char *dest)
 	char* temp_apvalue = NULL;
 #endif
 
-	while ((c = getopt(argc, argv, "D:E:I:l:m:M:N:q:r:R:u:U:g:G:H:W:")) != EOF) {
+	while ((c = getopt(argc, argv, "D:E:I:l:m:M:N:q:r:R:u:U:g:G:H:W:-:")) != EOF) {
```

Comment:
Shouldn't you be using getopt_long() for this? If you use getopt() then the following are equivalent:
pbs_rsub --hosts
pbs_rsub -- hosts

Reply:
'pbs_rsub -- hosts' returns directive error. OK, I will change it to getopt_long() anyway.

Reply:
This will be the first time we've added a "long" parameter in PBS Pro. I suspect we will start using them more often, so thank you for making the change.

Reply:
You can't use getopt_long().  That won't work on Windows.

Reply:
Do we need to implement getopt_long() for Windows or will we use the current solution?

Reply:
As per comments from @mike0042, we need not implement getopt_long(). But, have we tested these changes on Windows?

Reply:
No, I do not have any Windows machine. I probably could prepare some Windows testbed if necessary but it will take some time. I would appreciate help with the test on Windows.

Reply:
I don't want to force all this extra work on you @vchlum. I suggest we file a ticket to implement getopt_long() for Windows and refactor the existing getopt() calls in the future.